plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/info/SourceRole.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.info
+
+sealed class SourceRole {
+    data object Unknown : SourceRole()
```

Comment:
With `Unknown` also being an implementation of this, we have both `NoInfo` and `Wrapper(SourceRole.Unknown)` to indicate no information is available.  Is this intentional?